### DISCUSSION TITLE
Guidance from Experienced People!

### DISCUSSION THREAD

**Discussion by David-Braz** [community/community#147336] [Programming Help]
Hello, I’ve been wanting to start learning Spring Boot, so I decided to search the internet to find out what my first steps should be. I must admit I couldn’t reach any conclusions, so I decided to seek guidance from people who already have experience and know the best direction I should follow.
My experience with Java has been great so far. I’ve been adapting to Object-Oriented Programming and started to understand JPA a little.
I’d like to take a step forward with Spring Boot to create more robust systems and eventually work with Java Web.

**Thank you for your attention!**

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @David-Braz , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it to Programming Help for you. Good luck!

**Reply by David-Braz** [reply]
Thanks for the help.
